src/test/java/org/springframework/data/couchbase/repository/CouchbaseRepositoryQueryIntegrationTests.java

```diff
@@ -136,13 +141,19 @@ void findBySimpleProperty() {
 		Airport vie = null;
 		try {
 			vie = new Airport("airports::vie", "vie", "loww");
-			airportRepository.save(vie);
+			vie = airportRepository.save(vie);
 			List<Airport> airports = airportRepository.findAllByIata("vie");
-			assertEquals(vie.getId(), airports.get(0).getId());
+			assertEquals(1, airports.size());
+			System.out.println("findAllByIata(0): " + airports.get(0));
```

Comment:
are these printlns leftovers from debugging in the tests?

Reply:
removed